backend/crud/chat_history.py

```
@@ -0,0 +1,38 @@
+    Args:
+        db: 데이터베이스 세션
+        session_id: 세션 아이디
+        context: 채팅 히스토리
+    """
+    try:
+        stmt = insert(ChatHistory).values(session_id=session_id, context=context)
+        db.execute(stmt)
+        db.commit()
+    except Exception as e:
+        db.rollback()
+        raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)